index.html

```diff
@@ -24,22 +25,6 @@
                title="Bazel"
                alt="Bazel logo"
                class="logo img-responsive" />
-          <p class="tagline">{Fast, Correct} - Choose two</p>
```

Comment:
Unless we can come up with a different tagline that is at least equally memorable, concise, and distinctive, I prefer to keep this tagline here this way as it is since:

1) It does capture the fact that Bazel is both highly performant and correct, in contrast to other build systems that may be fast but are either not reproducible and/or require clean builds on a regular basis. The only key thing that it does not capture is that *Bazel is a build system* but that is already captured by the rest of the page.
2) The "Build and test software of any size, quickly and reliably" bit comes directly below, and the rest of the page is focused on Bazel's value prop, especially once we add illustrations to the value props below.
3) All other locations we have attempted to put this tagline to _not_ have it be the most prominent text on the page have resulted in the page appearing off-balance, and IMO the impact to the visual design of the page is worse than having the tagline as the hero title.